plugins/src/main/java/opengrok/auth/plugin/LdapAttrPlugin.java

```diff
@@ -126,13 +126,9 @@ String getSessionAllowedAttrName() {
 
     @Override
     public void fillSession(HttpServletRequest req, User user) {
-        boolean sessionAllowed;
-        LdapUser ldapUser;
-        Map<String, Set<String>> records = null;
-        Set<String> attributeValues;
-
         updateSession(req, false);
 
+        LdapUser ldapUser;
         if ((ldapUser = (LdapUser) req.getSession().
                 getAttribute(LdapUserPlugin.getSessionAttrName(ldapUserInstance))) == null) {
```

Comment:
I think it looks better like this:
```suggestion
        LdapUser ldapUser = (LdapUser) req.getSession().                getAttribute(LdapUserPlugin.getSessionAttrName(ldapUserInstance));
        if (ldapUser == null) {
```